config/config.go

```diff
@@ -216,6 +216,7 @@ type Config struct {
 	OidcSecretsFile                   string        `yaml:"oidc-secrets-file"`
 	OIDCCookieValidity                time.Duration `yaml:"oidc-cookie-validity"`
 	OidcDistributedClaimsTimeout      time.Duration `yaml:"oidc-distributed-claims-timeout"`
+	OIDCTokenCookieRemoveSubdomains   int           `yaml:"oidc-token-cookie-remove-subdomains"`
```

Comment:
Do we want `-token` part really? For OAuth Grant we ended up with it but here we have another cookie option `oidc-cookie-validity` so this could be `oidc-cookie-remove-subdomains` without implying any token stored in it. WDYT?

Reply:
That makes sense, the shorter the better. I removed the token from the name.